kyo-scheduler-zio/jvm/src/main/scala/kyo/KyoSchedulerRuntime.scala

```diff
@@ -0,0 +1,31 @@
+package kyo
+
+import kyo.scheduler.*
+import zio.*
+import zio.internal.ExecutionMetrics
+import zio.internal.FiberRuntime
+
+object KyoSchedulerRuntime {
+
+    lazy val default: Runtime[Any] = {
+        val exec =
+            new Executor {
+                val scheduler =
+                    kyo.scheduler.Scheduler.get
+                def metrics(implicit unsafe: Unsafe): Option[ExecutionMetrics] = None
+                def submit(runnable: Runnable)(implicit unsafe: Unsafe): Boolean = {
+                    scheduler.schedule(kyo.scheduler.Task(runnable.run()))
+                    true
+                }
+            }
+        Unsafe.unsafe { implicit u =>
+            Runtime.default.unsafe.run {
+                for {
+                    scope   <- Scope.make
+                    env     <- (Runtime.setExecutor(exec) ++ Runtime.setBlockingExecutor(exec)).build(scope)
```

Comment:
It should be safe to use global scope here:

https://github.com/zio/zio/blob/series/2.x/core/shared/src/main/scala/zio/Scope.scala#L130-L139

Reply:
updated